api/pkg/cli/formatter/field.go

```diff
@@ -52,3 +53,89 @@ func FormatTags(tags []*client.TagResponseBody) string {
 	}
 	return sb.String()
 }
+
+// FormatInfoDesc returns first 100 char of resource description
+func FormatInfoDesc(desc string) string {
```

Comment:
Why is this `specific` to description? Aren't all formatters supposed to be generic?

Reply:
Could you please add unit tests for all non-trivial formatters?